alpine/src/main/java/io/delta/alpine/data/RowParquetRecord.java

```diff
@@ -0,0 +1,25 @@
+package io.delta.alpine.data;
+
+import java.math.BigDecimal;
+import java.sql.Date;
+import java.sql.Timestamp;
+import java.util.List;
+import java.util.Map;
+
+public interface RowParquetRecord {
```

Comment:
How about just calling it `Row`? Technically, Delta can use other file formats (such as avro) for data files as well. Although I don't think we would change the format from parquet to other formats, it's still better to not expose `parquet` in the API.